src/bindings/python/src/pyopenvino/core/tensor.cpp

```diff
@@ -13,7 +13,7 @@
 namespace py = pybind11;
 
 void regclass_Tensor(py::module m) {
-    py::class_<ov::runtime::Tensor, std::shared_ptr<ov::runtime::Tensor>> cls(m, "Tensor");
+    py::class_<ov::Tensor, std::shared_ptr<ov::Tensor>> cls(m, "Tensor");
```

Comment:
on Python side - is 'openvino.runtime' also going to be replaced with 'openvino'?

Reply:
no

Reply:
not, unfortunately it's not going to be replaced right now.

CC @slyubimt